<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>students_db_main_window</class>
 <widget class="QMainWindow" name="students_db_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_DB">
    <property name="title">
     <string>&amp;DB</string>
    </property>
    <addaction name="act_Connect"/>
    <addaction name="act_Disconnect"/>
    <addaction name="separator"/>
    <addaction name="act_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_University">
    <property name="title">
     <string>&amp;University</string>
    </property>
    <addaction name="actView_departments"/>
    <addaction name="actView_students"/>
    <addaction name="actView_courses"/>
    <addaction name="actView_enrollments"/>
   </widget>
   <addaction name="menu_DB"/>
   <addaction name="menu_University"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_Connect">
   <property name="text">
    <string>&amp;Connect ...</string>
   </property>
  </action>
  <action name="act_Disconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="act_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actView_departments">
   <property name="text">
    <string>View departments...</string>
   </property>
  </action>
  <action name="actView_students">
   <property name="text">
    <string>View students...</string>
   </property>
  </action>
  <action name="actView_courses">
   <property name="text">
    <string>View courses...</string>
   </property>
  </action>
  <action name="actView_enrollments">
   <property name="text">
    <string>View enrollments...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
